<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UpdateProduit</class>
 <widget class="QDialog" name="UpdateProduit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>378</width>
    <height>197</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Update Produit</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>341</width>
     <height>161</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout"/>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>80</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>stock Initial</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnEditer">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>130</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Editer</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtId">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>30</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtStockInial">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>80</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="lblInfo">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>130</y>
     <width>181</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Id Produit</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
